content/02.introduction.md

```diff
@@ -121,7 +121,24 @@ The mechanism behind these insults has been suggested to be related to inflammat
 Individual outcomes have varied significantly, and reported hospitalization rates have been wide-ranging based on age and location, with various underlying health conditions being disproportionately reported [@doi:10.15585/mmwr.mm6915e3].
 Outcomes in patients that are hospitalized have generally been poor, with rates of admission to the intensive care unit (ICU) upwards of 15% in both Wuhan, China and Italy [@doi:10.1056/nejmoa2002032; @doi:10.1001/jama.2020.2648; @doi:10.1001/jama.2020.4031].
 
-Though older populations are generally considered the most vulnerable, pediatric infections are also a concern.
+#### Subpopulations of Special Concern  
+
+The prevalence of different clinical features of COVID-19 infection differ in different areas of the world, presumably due to differential reporting and/or different levels of pre-existing risk factors [@doi:10.1007/s00592-020-01546-0].
+In the context of the United States, persons infected with COVID-19 are more likely to require hospitalization if they are of male sex, older age, and/or of Black/African American background [@doi:10.1377/hlthaff.2020.00598; @doi:10.15585/mmwr.mm6925e1].
```

Comment:
I would like to think about this more, but for now just noting: The wording here might be changed here to make more rigid what's more confirmed or ~rigid and more flexible what should be more flexible. Here's what I mean. It could start instead with the main risk factors that are put by the CDC (age, co-morbidities / disease conditions that could lead to higher risk -- see whatever is the most up to date). Then separately start with a sentence that also describes other possible patterns (not really risk factors, just patterns) seen in the data "so far" -- I say "patterns" because I'm guessing there's still analysis being done and because it's might be more complicated or different to figure out why a pattern related to ~ethnic background or ~race may be seen versus a pattern related to a co-morbidity (although this is also complicated) or age (again although this is also complicated) & I say "so far" is because these patterns might be partially changed if some of the reasons for the patterns are adjustable (e.g. if there's a lack of access --> increase access) and actually all the patterns might be changed because public health sometimes adds targeting (or improves coverage) to populations identified through data as more vulnerable (or more generally fills in coverage gaps).

Reply:
Thank you, @yemarshall !  One thing we could do is include a few sentences to say that in theory there may be different ways in which a group could have higher risk for COVID-19 deaths or other severe outcomes.  They could:
1.  be more likely to contact the virus (e.g., essential workers, people living in big cities?)
2.  be more likely to get infected once exposed (e.g., depressed immune system?)
3.  be more likely to get seriously sick once infected (e.g., comorbidities, older age?)
4.  be less likely to get adequate help once seriously sick (e.g., poor, minorities, far from health care).
These can all get lumped together because some of the steps aren't easy to empirically study (e.g., it seems hard to study #2 directly without doing some really horrible human challenge study, and by definition of #4 the people aren't around to be studied by the scientists).  So in practice they add up to the same thing, but in terms of mechanisms they are completely different.  In a few cases the distinction might be especially important (e.g., is a group less resistant to the virus for genetic reasons, or are they living in more crowded places, or are they not getting enough information, or not enough care -- if we knew for sure which one, it would in principle be easier to design an intervention).   Of course we could never resolve this in one subsection in a literature review, but maybe it would be a good contribution to list the questions?  What do you think, @yemarshall and @rando2?

Reply:
@dziakj1 I like that breakdown! Probably outside the scope of this section, but there is also a loop from 4 to 2 due to allostatic load, i.e. the long-term burden of chronic stress -- assuming that reduced to access to health resources is correlated with other life stressors such as experiencing racism or poverty.

Potentially framing it as something along the lines of:
There are some factors that are associated with greater risk from COVID-19. There are several possible etiologies through which a factor could influence COVID-19 outcomes, which could (your list of 4).

(I am not really sure if this is the best way to phrase this because I am not a public health person!)